CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ All notable changes to this project will be documented in this file.
 - Docs: Revisit specification of \_near relation [(#563)](https://github.com/sakuli/sakuli/issues/563)
 - Maintenance: Add windows e2e tests to Sakuli [(#580)](https://github.com/sakuli/sakuli/issues/580)
 - Bugfix: Dynamic wait not initialised, reason: NoSuchSessionError [(#583)](https://github.com/sakuli/sakuli/issues/583)
+- Enhancement: improve confidence of scrollIntoViewIfNeeded [(#482)](https://github.com/sakuli/sakuli/issues/482)
```

Comment:
As the issue was not located in scroll into view. Could you please rename the issue?
It is also not required to be put into changelogs as this is just a CI change now, which is not affecting the delivered product at all.